### PR TITLE
docs(flutter_infinite_list): update documentation

### PR DESCRIPTION
**Pull request by Gene-Dana**
## Status

**READY**

## Breaking Changes

NO

## Description

docs(flutter_infinite_list): update documentation - closes: #2341

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: docs/flutterinfinitelisttutorial.md [hunk 1]
```diff
@@ -20,15 +19,50 @@
...
 
-[script](_snippets/flutter_infinite_list_tutorial/flutter_create.sh.md ':include')
+```sh
+flutter create flutter_infinite_list
+```
 
 We can then go ahead and replace the contents of pubspec.yaml with
 
-[pubspec.yaml](_snippets/flutter_infinite_list_tutorial/pubspec.yaml.md ':include')
+[pubspec.yaml](https://raw.githubusercontent.com/felangel/bloc/master/examples/flutter_infinite_list/pubspec.yaml ':include')
```
Comment (felangel) [inline docs/flutterinfinitelisttutorial.md:28]: awesome! Can we delete the old snippet for the pubspec.yaml?

Comment (Gene-Dana) [reply docs/flutterinfinitelisttutorial.md:28]: Yes we can, and we can do this update for all the relevant snippets in the tutorials

Comment (Gene-Dana) [reply docs/flutterinfinitelisttutorial.md:28]: Removed all the old snippets from this tutorial

File: docs/flutterinfinitelisttutorial.md [hunk 2]
```diff
@@ -125,41 +154,46 @@ One optimization we can make is to `debounce` the `Events` in order to prevent s
...
-Whenever the user scrolls, we calculate how far away from the bottom of the page they are and if the distance is ≤ our `_scrollThreshold` we add a `PostFetched` event in order to load more posts.
+Whenever the user scrolls, we calculate how far you have scrolled down the page and if our distance is ≥ 90% of our `maxScrollextent` we add a `PostFetched` event in order to load more posts.
 
 Next, we need to implement our `BottomLoader` widget which will indicate to the user that we are loading more posts.
 
-[bottom_loader.dart](_snippets/flutter_infinite_list_tutorial/bottom_loader.dart.md ':include')
+[bottom_loader.dart](https://raw.githubusercontent.com/felangel/bloc/master/examples/flutter_infinite_list/lib/posts/widgets/bottom_loader.dart ':include')
 
-Lastly, we need to implement our `PostWidget` which will render an individual Post.
+Lastly, we need to implement our `PostListIten` which will render an individual Post.
```
Comment (felangel) [inline docs/flutterinfinitelisttutorial.md:194]:
```suggestion
Lastly, we need to implement our `PostListItem` which will render an individual Post.
```

File: docs/flutterinfinitelisttutorial.md [hunk 3]
```diff
@@ -125,41 +154,46 @@ One optimization we can make is to `debounce` the `Events` in order to prevent s
...
 
-[post.dart](_snippets/flutter_infinite_list_tutorial/post_widget.dart.md ':include')
+[post_list_iten.dart](https://raw.githubusercontent.com/felangel/bloc/master/examples/flutter_infinite_list/lib/posts/widgets/post_list_item.dart ':include')
```
Comment (felangel) [inline docs/flutterinfinitelisttutorial.md:196]:
```suggestion
[post_list_item.dart](https://raw.githubusercontent.com/felangel/bloc/master/examples/flutter_infinite_list/lib/posts/widgets/post_list_item.dart ':include')
```

File: docs/flutterinfinitelisttutorial.md [hunk 4]
```diff
@@ -20,15 +19,50 @@
...
+|   |   |   └── post_state.dart
+|   |   └── models
+|   |   |   └── models.dart*
+|   |   |   └── post.dart
+│   │   └── view
+│   │   |   ├── posts_page.dart
+│   │   |   └── posts_list.dart
+|   |   |   └── view.dart*
+|   |   └── widgets
+|   |   |   └── bottomloader.dart
```
Comment (felangel) [inline docs/flutterinfinitelisttutorial.md:53]:
```suggestion
|   |   |   └── bottom_loader.dart
```

File: docs/flutterinfinitelisttutorial.md [hunk 5]
```diff
@@ -173,20 +207,16 @@ Even though in this application we only have one bloc, it's fairly common in lar
 
 If we want to be able to do something in response to all `Transitions` we can simply create our own `BlocObserver`.
 
-[simple_bloc_observer.dart](_snippets/flutter_infinite_list_tutorial/simple_bloc_observer.dart.md ':include')
+[simple_bloc_observer.dart](https://raw.githubusercontent.com/felangel/bloc/master/examples/flutter_infinite_list/lib/simple_bloc_observer.dart':include')
```
Comment (felangel) [inline docs/flutterinfinitelisttutorial.md:210]:
```suggestion
[simple_bloc_observer.dart](https://raw.githubusercontent.com/felangel/bloc/master/examples/flutter_infinite_list/lib/simple_bloc_observer.dart ':include')
```

**Review by felangel — APPROVED** [review]
LGTM 💯 🚀

**Pull request merged**